usaspending_api/api_docs/markdown/request_recipes.md

```diff
@@ -0,0 +1,75 @@
+<ul class="nav nav-stacked" id="sidebar">
+  <li><a href="#award-recipes">Award Recipes</a></li>
+</ul>
+[//]: # (Begin Content)
+
+# Award Recipes <a name="award-recipes"></a>
+
+These example requests are for the `/api/v1/awards/` endpoint.
+
```

Comment:
I think these requests need detailed descriptions of what you're querying for. For example, instead of `all rewards for a specific agency`, something like:
```
Find all awards for that were awarded by agency XXXX, which has a CGAC code of 0971 and is the toptier agency.
```